AppLovin MAX Demo App - ObjC/AppLovin MAX Demo App - ObjC.xcodeproj/project.pbxproj

```diff
@@ -10,6 +10,7 @@
 		1D992FE6231FA1C400C472F8 /* Main.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 1D992FE4231FA1C400C472F8 /* Main.storyboard */; };
 		1D992FE8231FA1C500C472F8 /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 1D992FE7231FA1C500C472F8 /* Assets.xcassets */; };
 		1D992FEB231FA1C500C472F8 /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 1D992FE9231FA1C500C472F8 /* LaunchScreen.storyboard */; };
+		2C1195B2CD283602C5035D53 /* Pods_Non_Mediated_Projects_AppLovin_MAX_Demo_App___ObjC.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 372F2921225A0A5DF89CAB0E /* Pods_Non_Mediated_Projects_AppLovin_MAX_Demo_App___ObjC.framework */; };
```

Comment:
Let's not check in CocoaPods changes since pubs are expected to do a fresh install.

Reply:
You can de-integrate CocoaPods from the project for a "fresh" `.xcodeproj` or manually leave out the lines before checking them in.